Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -404,7 +404,8 @@ - (void)updateFeedSourcesUserInitiated:(BOOL)wasUserInitiated completion:(nonnul
     }
     self.loadingNewContent = YES;
     if (!self.refreshControl.isRefreshing) {
-        [self.refreshControl beginRefreshing];
+        // TODO: fix this - causing UITest to hang for a few minutes. See https://stackoverflow.com/questions/40983184/xcode-8-ui-testing-taking-very-long
```

Comment:
This needs to be fixed before merging.

Reply:
Fixed! See https://github.com/wikimedia/wikipedia-ios/pull/2314/commits/7ec6fef6288e416d58ffde8cb5f76138789df9da